org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,272 @@
+app-id: org.tenacityaudio.Tenacity
+default-branch: stable
```

Comment:
```suggestion
```

It's the default-branch when build by flathub, and get overridden when needed.

Otherwise it's just a nuisance when building locally.